docs/content/guides/dagster/software-defined-assets.mdx

```diff
@@ -63,29 +63,45 @@ The framework infers asset dependencies by looking at the names of the arguments
 
 ### Connecting assets to external services
 
-Having defined some assets, we can combine them with resources and IO managers to determine how they're stored, and connect them to external services. We use <PyObject object="with_resources" /> to provide resources to assets and source assets.
+Having defined some assets, we can combine them with resources, such as IO managers, to determine how they're stored, and connect them to external services.
 
-It's common to use a utility like <PyObject object="load_assets_from_modules" /> or <PyObject object="load_assets_from_package_name" /> to pick up all the assets within a module or package, so you don't need to list them individually.
+We use <PyObject object="Definitions" /> to load definitions such as assets and resources. Let's take a closer look at how definitions are loaded:
 
-```python file=../../assets_pandas_pyspark/assets_pandas_pyspark/assets/weather_assets.py startafter=gather_assets_start endbefore=gather_assets_end
-# imports the module called "assets" from the package containing the current module
-# the "assets" module contains the asset definitions
-from . import table_assets
-from dagster import load_assets_from_modules, with_resources
+- We add the assets that we want Dagster to load in the `assets` argument to the <PyObject object="Definitions" /> object. It's common to use a utility like <PyObject object="load_assets_from_modules" /> or <PyObject object="load_assets_from_package_name" /> to pick up all the assets within a module or package, so you don't need to list them individually. The order that we supply the assets doesn't matter, since the dependencies are determined by each asset definition.
 
-weather_assets = with_resources(
-    load_assets_from_modules(modules=[table_assets]),
-    resource_defs={
-        "io_manager": IOManagerDefinition.hardcoded_io_manager(LocalFileSystemIOManager())
-    },
-)
-```
+- We supply resources mapped to the assets using the `resources` argument to the <PyObject object="Definitions" /> object.
+
+```python file=../../assets_pandas_pyspark/assets_pandas_pyspark/__init__.py startafter=gather_assets_start endbefore=gather_assets_end
+# __init__.py
+
+    # imports the module called "assets" from the package containing the current module
```

Comment:
This comment is above the wrong line

Reply:
that part is fine. but `    # imports the module called "assets" from the package containing the current module` could be above the line that imports the assets module.